src/js/App/Header/ContextSwitcher.js

```diff
@@ -12,6 +13,15 @@ const ContextSwitcher = ({ user }) => {
   const onSelect = () => {
     dispatch(onToggleContextSwitcher());
   };
+  const dropdownItems = [
+    <DropdownItem key="link" description="PERSONAL ACCOUNT">
+      {user.identity.account_number}
+    </DropdownItem>,
+    <DropdownItem key="link">678909</DropdownItem>,
+    <DropdownItem key="link">678735</DropdownItem>,
+    <DropdownItem key="link">123456</DropdownItem>,
+  ];
+
   return (
     <Dropdown
       className="ins-c-page__context-switcher-dropdown"
```

Comment:
can you add the prop `ouiaId="Account Switcher"`? This will make the dropdown easily model-able in our automation.